src/main/java/Pokemon.java

```diff
@@ -0,0 +1,57 @@
+public abstract class Pokemon {
+    private String name;
+    private int level;
+    private String pokemonType;
+
+//    constructor
```

Comment:
Dit soort methodes is misschien handig voor je eigen overzicht, maar wanneer je het op github zet wil je eigenlijk weg halen. Comments zijn nuttig wanneer ze iets toevoegen wat niet anders gecommuniceerd kan worden en als dat zo is, moet je bij jezelf nadenken waarom dit niet anders gecommuniceerd kan worden en of er dan niet iets fout is. In dit geval kunnen we al zien dat het een constructor is, dus is de comment overbodig.